src/App.tsx

```diff
@@ -15,48 +16,128 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort((g1, g2) => g1.localeCompare(g2));
```

Comment:
```suggestion
    visibleGoods.sort((g1, g2) => g1.localeCompare(g2));
```
either name `g1 and g2` as `a and b`, or better to use `good1 and good2`

Reply:
done